src/common/POLFields/FieldsLocation/FieldsLocation.js

```diff
@@ -36,19 +37,30 @@ const FieldsLocation = ({
   locationIds,
   locations,
   pOLineFormValues: { orderFormat, physical, eresource, instanceId } = {},
+  poNumber,
   withValidation,
 }) => {
   if (!locations) return null;
 
   const isPhysicalQuantityRequired = isLocationPhysicalQuantityRequired(orderFormat, physical?.createInventory);
   const isElectronicQuantityRequired = isLocationEresourceQuantityRequired(orderFormat, eresource?.createInventory);
   const validate = withValidation ? validateLocation : NO_VALIDATE;
+  const receivingLink = {
```

Comment:
IMO it's better to make a string `/rec?qindex` instead of having new object each time